spec/features/user_sessions_spec.rb

```diff
@@ -100,4 +100,16 @@
     click_button 'Next: Add Garden'
     expect(page).to have_content('Your Gardens')
   end
+
+  it 'should redirect to sign up page when user is not authorized' do
+    visit new_crop_path
+    see("You're not authorized to go to there.")
+    user.password = 'password123'
+    user.password_confirmation = 'password123'
+    user.save
+    fill_in :user_email, with: user[:email]
+    fill_in :user_password, with: 'password123'
+    click_button "Sign in"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.